Jenkinsfile

```diff
@@ -47,8 +47,14 @@ test ${TARGET_URL} --location us-east-1-linux:Chrome%20Canary --bodies --keepua
           stash includes: 'wpt.json', name: 'wpt.json'
         }
         failure {
-          ircNotification()
-        }
+          ircNotification('#perftest-alerts')
+          emailext(
+                  attachLog: true,
+                  body: '$BUILD_URL\n\n$FAILED_TESTS',
+                  replyTo: '$DEFAULT_REPLYTO',
+                  subject: '$DEFAULT_SUBJECT',
+                  to: '$DEFAULT_RECIPIENTS')
```

Comment:
Could you reduce the indent to 2 spaces, before merging? Thanks!

Reply:
Sure!